atom-client/README.md

```diff
@@ -0,0 +1,9 @@
+Atom Client Module
+===
```

Comment:
Oh right. I hate the need for that, so I preceded the title by a # instead, for a similar effect.